### DISCUSSION TITLE
bytesReceived and bytesSent stay at 0 even if data is transported using the media transport (video)

### DISCUSSION THREAD

**Discussion by oanguenot** [paullouisageneau/libdatachannel#1138] [Q&A]
Hi,

[Question discussed from node-datachannel]
I use node-datachannel and when transporting a video track, I saw that when calling the function bytesReceived() or bytesSent(),  the value stays at 0. Only messages exchanged through the data-channel are taken into account.

Is it possible to have access to statistics regarding the media transport ? 
- bytesSent, bytesReceived
- PacketsSent, packetsReceived, packetsLost
- ...

Thanks in advance

**Comment by paullouisageneau** [comment]
There are currently no statistics for the media transport sadly, but I'd be good to add it.
